distribution/archives/integ-test-zip/build.gradle

```diff
@@ -17,56 +17,7 @@
  * under the License.
  */
 
```

Comment:
I think it is worth adding a description or a comment explaining what this project is for. Also it'd be nice to explain that the vast majority of its build is configured in the parent project. I mean, it is obvious to me when I see a nearly empty gradle file that I have to look up one level, but we may as well make it more obvious.